src/i18n/languages/de_DE.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * @preserve
+ * Authors: Stefan Salzl
+ * Last updated: Dec 12, 2017
+ *
+ * Description: Definition file for German - Switzerland language-country.

```

Comment:
Please change part of description from `German - Switzerland` to `German - Germany`.